### PR TITLE
improved README.md for jumpstart and added LICENCE.md

### PR DESCRIPTION
**Pull request by rogdevil**
upon the suggestion of Bupesh I have improved README.md added a licence and added the basic contributing guidelines for any potential contributor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
+Jumpstart is a versatile code template generator that allows you to kickstart your projects with ease.  Simply follow the instructions below to get started. 
+##### *It is actively under development to support different frameworks and languages, for updates please star the repository
 
-## Opinions
-
-- Github repos get configured with ssh auth for push
-- Uses Github repo naming convention for namespacing / orgs
-
-## Development
+## Development -
```
Comment (bhupeshisanurd) [inline README.md]:
```suggestion
## Development
```

File: README.md [hunk 2]
```diff
@@ -16,3 +12,50 @@ Build for your local architecture
 ```
 make releases/jumpstart
 ```
+## Getting started -
```
Comment (bhupeshisanurd) [inline README.md]:
```suggestion
## Getting started
```

File: README.md [hunk 3]
```diff
@@ -16,3 +12,50 @@ Build for your local architecture
...
+Create a GitHub personal access token and set it as a variable GITHUB_TOKEN in your terminal profile.
+```bash
+export GITHUB_TOKEN=<your_token>
+```
+#### 4. Generate Your Project
+Run the following command to generate a project:
+```bash
+releases/jumpstart -t <template name> <project name>
+```
+This command will create a GitHub repository with the name ```<project name>``` and the chosen template ```<template name>```. Navigate into your project directory using:
```
Comment (bhupeshisanurd) [inline README.md]:
```suggestion
This command will create a private GitHub repository with the name ```<project name>``` and the chosen template ```<template name>```. Navigate into your project directory using:
```

File: README.md [hunk 4]
```diff
@@ -1,13 +1,9 @@
-# jumpstart
+# Jumpstart
 
-Project generator for various languages and frameworks
+Jumpstart is a versatile code template generator that allows you to kickstart your projects with ease.  Simply follow the instructions below to get started. 
+##### *It is actively under development to support different frameworks and languages, for updates please star the repository
```
Comment (bhupeshisanurd) [inline README.md]:
```suggestion
> [!IMPORTANT]  
> Jumpstart is under active development to support different frameworks and languages, for updates please star (or watch) the repository.
```

Comment (bhupeshisanurd) [reply README.md]: For reference: https://blog.jakelee.co.uk/github-alert-experiments/

File: LICENCE.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Released under MIT License
+
+Copyright (c) 2013 Mark Otto.
+
+Copyright (c) 2017 Andrew Fong.
```
Comment (bhupeshisanurd) [inline LICENCE.md]: Who are these people? please add nurdsoft here. Also update the year to 2023.

File: README.md [hunk 6]
```diff
@@ -1,13 +1,9 @@
-# jumpstart
+# Jumpstart
 
-Project generator for various languages and frameworks
+Jumpstart is a versatile code template generator that allows you to kickstart your projects with ease.  Simply follow the instructions below to get started.
```
Comment (bhupeshisanurd) [inline README.md]:
```suggestion
Jumpstart is a versatile code template generator that allows you to kickstart your projects with ease.
```

A one-liner project description is good, devs are aware that the rest of the readme will contain setup/build instrutions.

File: README.md [hunk 7]
```diff
@@ -16,3 +12,50 @@ Build for your local architecture
...
+## Getting started -
+
+#### 1. Clone the Repository
+
+```bash
+git clone https://github.com/nurdsoft/jumpstart
+```
+#### 2. Build for Local Machine
+
+Navigate to the cloned repository and build the tool for your local machine using the following command:
```
Comment (bhupeshisanurd) [inline README.md]:
```suggestion
Navigate to the cloned repository and build the tool for your machine using the following command:
```

File: README.md [hunk 8]
```diff
@@ -16,3 +12,50 @@ Build for your local architecture
...
+
+#### 3. Set Up GitHub Personal Access Token
+Create a GitHub personal access token and set it as a variable GITHUB_TOKEN in your terminal profile.
+```bash
+export GITHUB_TOKEN=<your_token>
+```
+#### 4. Generate Your Project
+Run the following command to generate a project:
+```bash
+releases/jumpstart -t <template name> <project name>
```
Comment (bhupeshisanurd) [inline README.md:43]: How do we find what are the available templates? Is there a command for it? if yes, please list that before the generate command.
If such a command doesn't exist, we need to list all the "exact" template names somewhere.

**Review by bhupeshisanurd — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rogdevil) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (rogdevil) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (rogdevil) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (rogdevil) [reply README.md]: done

File: LICENCE.md (hunk 5 above)
Comment (rogdevil) [reply LICENCE.md]: I copied this licence text from a MIT site link I didn't knew that we can change the copyright text as well sorry : |
https://www.mit.edu/~amini/LICENSE.md
done

File: README.md (hunk 6 above)
Comment (rogdevil) [reply README.md]: done

File: README.md (hunk 7 above)
Comment (rogdevil) [reply README.md]: done

File: README.md (hunk 8 above)
Comment (rogdevil) [reply README.md:43]: done

File: README.md [hunk 9]
```diff
@@ -16,3 +13,55 @@ Build for your local architecture
...
+```
+#### 2. Build for Local Machine
+
+Navigate to the cloned repository and build the tool for your machine using the following command:
+```bash
+make releases/jumpstart
+```
+
+#### 3. Set Up GitHub Personal Access Token
+Create a GitHub personal access token and set it as a variable GITHUB_TOKEN in your terminal profile.
```
Comment (bhupeshisanurd) [inline README.md]:
```suggestion
Create a GitHub personal access token with read/write repo access and set it as a variable GITHUB_TOKEN in your terminal profile.
```

**Review by bhupeshisanurd — CHANGES_REQUESTED** [review]

File: README.md (hunk 9 above)
Comment (rogdevil) [reply README.md]: done

**Review by bhupeshisanurd — APPROVED** [review]

**Pull request merged**